x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/metric_item/metric_item_icon.tsx

```diff
@@ -43,35 +44,31 @@ const Container = styled.div`
 export const MetricItemIcon = ({
   monitor,
   status,
-  ping,
   timestamp,
   configIdByLocation,
 }: {
   monitor: OverviewStatusMetaData;
   status: string;
   configIdByLocation: string;
   timestamp?: string;
-  ping?: OverviewPing;
 }) => {
   const testNowRun = useSelector(manualTestRunSelector(monitor.configId));
   const isPopoverOpen = useSelector(selectErrorPopoverState);
+  const { latestPing } = useLatestError({
```

Comment:
I think we should either hack this with local state to not be `undefined` during load, or fix it upstream so it doesn't reset it during load but only on completed.

It makes this popover really unpleasant during auto-refreshing of the data, and if the user's mouse is too far up the popover it will suddenly trigger `onMouseLeave` and _close_ the popover even if the user doesn't want it to close.

![20250219111745](https://github.com/user-attachments/assets/1bf0b2bb-b5ef-4a37-8926-d594c9575801)